### DISCUSSION TITLE
ER diagram of laravel-permission tables

### DISCUSSION THREAD

**Discussion by muath-ye** [spatie/laravel-permission#2292] [Q&A] [answered]
It is not a question but I need if some one has the ER diagram of the tables of this package

**Comment by drbyte** [comment, marked as answer]
In the Docs, under **Advanced -> Other** is a link to an ER diagram.

https://spatie.be/docs/laravel-permission/v5/advanced-usage/other

**Comment by muath-ye** [comment]
Thank you @drbyte 
Here is the redirected link
https://drawsql.app/templates/laravel-permission

And here is the image of ER diagram:
![image](https://user-images.githubusercontent.com/34031333/210503672-878767de-616b-473e-8bd8-8b37a249b95b.png)
